papis/git.py

```diff
@@ -9,6 +9,13 @@
 logger = papis.logging.get_logger(__name__)
 
 
+def init(path: str) -> None:
+    """Initialize a git repository at *path*."""
+
+    logger.info("Initializing git reposityro: '%s'.", path)
```

Comment:
```suggestion
    logger.info("Initializing git repository: '%s'.", path)
```

Reply:
Nice catch! Fixed in the rebase https://github.com/papis/papis/compare/56d753008eccad123bcfba9ad53a2686edff89a5..8b16b28d0e04e7d69d5916210d4438d2dd708ae9.